src/ts/sign.ts

```diff
@@ -165,6 +200,29 @@ export async function signOrder(
   };
 }
 
+/**
+ * Returns the signature for the Order Cancellation with the signing scheme encoded
+ * into the signature bytes.
+ *
+ * @param domain The domain to sign the cancellation.
+ * @param cancellation The cancellation to be signed.
+ * @param owner The owner for the order used to sign.
+ * @param scheme The signing scheme to use. See {@link SigningScheme} for more
+ * details.
+ * @return Encoded signature including signing scheme for the cancellation.
+ */
+export async function signOrderCancellation(
+  domain: TypedDataDomain,
+  cancellation: OrderCancellation,
```

Comment:
maybe it's slightly more convenient for the caller to directly pass the UID here?